src/chm/documents/customactions/wixsettingchange.html.md

```diff
@@ -0,0 +1,37 @@
+---
+title: WixBroadcastSettingChange and WixBroadcastEnvironmentChange Custom Actions
+layout: documentation
+after: using_standard_customactions
+---
+
+# WixBroadcastSettingChange and WixBroadcastEnvironmentChange Custom Actions
+
+The WixBroadcastSettingChange and WixBroadcastEnvironmentChange custom actions are immediate custom actions that send a WM\_SETTINGCHANGE message to all top-level windows indicating that settings have changed. WixBroadcastSettingChange indicates that unspecified settings have changed. WixBroadcastEnvironmentChange indicates that environment variables have changed.
+
+Other programs are supposed to listen for WM\_SETTINGCHANGE and update any internal state with the new setting.
```

Comment:
I wouldn't say "are supposed to". I would say something like "Other programs can listen...". They don't have to if they don't care. Most don't handle it.